### PR TITLE
added defer attribute

### LINKED ISSUE #92: Add 'defer' attributes to script tags in index.ejs
Having all the external JavaScript loaded before the HTML means that on a slow connection the page will not display until all of the resources are downloaded, but we don't really need the JavaScript to be able to render the keyboard, only to start the welcome animation.

For each of the `<script>` tags at the top of the HTML body, please add a `defer` attribute to them.

After making this change, please test that the page still works correctly. You can simulate slow loading times to help with [Chrome Dev Tools throttling](https://helpdeskgeek.com/networking/simulate-slow-internet-connection-testing/).

For more background context, see [async vs defer](https://www.growingwiththeweb.com/2014/02/async-vs-defer-attributes.html).

### PR DESCRIPTION
**Pull request by racheltl**
Fixes #92

### CODE REVIEW

**Comment by apoorva-mk** [comment]
Hi, this is file is to be deleted.

File: userinfo.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
 </head>
 <body>
     <p>hello</p>
-    <script src="routes/routes.js"></script> 
+    <script src="routes/routes.js" defer></script>
```
Comment (apoorva-mk) [inline userinfo.html:10]: This file will be deleted as there is an open PR which will do so.

Comment (alodahl) [reply userinfo.html:10]: I've asked @vegetabill for this thoughts on this... he'll be back from lunch in a bit.

**Review by vegetabill — CHANGES_REQUESTED** [review]
@racheltl sorry for the confusion but this file is not actually used in the project which is why it's being deleted.

The issue was to update the main HTML file used by the app, which is actually found at `views/index.ejs`

**Comment by vegetabill** [comment]
Oh nvm you already opened #113 to fix the correct files. Closing this as it can't be merged and isn't needed. If you were trying to proactively fix all the HTML, thanks! Sorry it was an unused file.

**Pull request closed without merging**